### ISSUE TITLE
No detail chapter on shiny prerendered. Should there be one ? 

### ISSUE THREAD

**Issue by cderv** [rstudio/rmarkdown-book#37]
Looking for detailed information on `runtime:shiny_prerendered` I just saw that the book does not explain this format of the rmarkdown 📦 . It is only mentioned in the _Interactive Tutorial_ part, as something to use with `learnr::tutorial`. 

I think this is a missing part of the rmarkdown 📦 in the reference Rmarkdown book. Something that should be added in another edition or in another book i.e advanced Rmarkdown.

**Comment by yihui** [comment]
Yes, we should add it at some point. I avoided it in the current edition because I felt it was not quite ready---there was an issue that could easily confuse users (to be fixed via https://github.com/rstudio/rmarkdown/pull/1420, which I haven't reviewed or merged yet).

Thanks for the suggestion!
